cpp/core/quadrature.cpp

```diff
@@ -533,12 +589,42 @@ quadrature::compute_gauss_jacobi_rule(double a, int m)
 }
 //-----------------------------------------------------------------------------
 std::pair<Eigen::ArrayXd, Eigen::ArrayXd>
+quadrature::compute_gll_rule(double a, int m)
```

Comment:
What is "a"?

Reply:
I was using the `compute_gauss_jacobi_rule` as an example and forgot to remove "a". I'll remove in them now.